### ISSUE TITLE
Tried to install, pip was not happy.

### ISSUE THREAD

**Issue by krestivo-altoslabs** [glencoesoftware/omero-user-token#4]
```
 pip install -e .
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///home/cust/altos/src/omero-user-token
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [12 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/home/cust/altos/src/omero-user-token/setup.py", line 49, in <module>
          version=version.getVersion(),
        File "/home/cust/altos/src/omero-user-token/version.py", line 114, in getVersion
          version = readGitVersion() or release_version
        File "/home/cust/altos/src/omero-user-token/version.py", line 77, in readGitVersion
          m = re.search(_GIT_DESCRIPTION_RE, ver)
        File "/usr/lib/python3.9/re.py", line 201, in search
          return _compile(pattern, flags).search(string)
      TypeError: cannot use a string pattern on a bytes-like object
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by krestivo-altoslabs** [comment]
Also tried 

```
 python setup.py build
Traceback (most recent call last):
  File "/home/cust/altos/src/omero-user-token/setup.py", line 49, in <module>
    version=version.getVersion(),
  File "/home/cust/altos/src/omero-user-token/version.py", line 114, in getVersion
    version = readGitVersion() or release_version
  File "/home/cust/altos/src/omero-user-token/version.py", line 77, in readGitVersion
    m = re.search(_GIT_DESCRIPTION_RE, ver)
  File "/usr/lib/python3.9/re.py", line 201, in search
    return _compile(pattern, flags).search(string)
```
based on guessing from https://github.com/glencoesoftware/omero-user-token/blob/master/appveyor.yml#L10

**Comment by sbesson** [comment]
@krestivo-altoslabs if you are interested in the latest release, you might want to install the wheel available as a release asset at https://github.com/glencoesoftware/omero-user-token/releases/tag/v0.1.1.

Otherwise, for building from source with modern versions of Python 3.x, you might need the fix proposed in #3 which we will review for inclusion

**Comment by krestivo-altoslabs** [comment]
Thanks!
